### PR TITLE
Tidy up static FL DB

### LINKED ISSUE #624: Variety of small changes to the static FL DB
- [ ] Remove all double underscores from table names
- [ ] Prefix all tables with "static_"
- [ ] Change all references of "fl_process_id" to be integer, not bigint
- [ ] Change "cycle_id" in static_worker_cycle to be an integer, not bigint
- [ ] Change "sequence" on static_cycle to be an integer
- [ ] Add "value_tfjs" to protocol as largebinary(blob)
- [ ] Change "model_id" on static_model_checkpoint to be an integer, not a varchar
- [ ] Change "ping", "avg_download", and "avg_upload" from bigint to float
- [ ] Change "values" to "value" on static_model_checkpoint
- [ ] Remove "format_preference" on static_worker

Note, that for some of these changes, you'll also need to change all references to this DB column throughout the PyGrid application. This won't be as simple as changing the column names, unfortunately.

### LINKED ISSUE #627: Add base mixins to SQLAlchemy
We want to add the following to SQLAlchemy:

https://github.com/absent1706/sqlalchemy-mixins

Ensure that we hook this with `flask_sqlalchemy` appropriately and that ALL models that we have in the database inherit from this base mixin. While this will add a lot of features that we can convert to over time, it will specifically add `created_at` and `updated_at` fields to all models and manage them without us needing to.

### PR DESCRIPTION
**Pull request by Benardi**
## Description

Fixes #624 
Fixes #627 

- [x] Remove all double underscores from table names
- [x] Prefix all tables with "static_"
- [x] Change all references of "fl_process_id" to be integer, not bigint
- [x] Change "cycle_id" in static_worker_cycle to be an integer, not bigint
- [x] Change "sequence" on static_cycle to be an integer
- [x] Add "value_tfjs" to protocol as largebinary(blob)
- [x] Change "model_id" on static_model_checkpoint to be an integer, not a varchar
- [x] Change "ping", "avg_download", and "avg_upload" from bigint to float
- [x] Change "values" to "value" on static_model_checkpoint
- [x] Remove "format_preference" on static_worker

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

**Comment by Benardi** [comment]
> Hey @Benardi, could you please _remove all double underscores from table names_? as mentioned on #624. :)

Sure thing, I think it would be best if this PR were repurposed to fix the issue you mentioned.

**Comment by hericlesme** [comment]
Awesome! 🎉

File: grid/app/main/sfl/cycles/cycle.py [hunk 1]
```diff
@@ -14,15 +14,15 @@ class Cycle(db.Model):
         fl_process_id (Integer,ForeignKey): Federated learning ID that owns this cycle.
     """
 
-    __tablename__ = "__cycle__"
+    __tablename__ = "static_cycle_"
```
Comment (IonesioJunior) [inline grid/app/main/sfl/cycles/cycle.py:17]: Why are we keeping one `_` after table names?

Comment (Benardi) [reply grid/app/main/sfl/cycles/cycle.py:17]: No particular reason, it simply wasn't specified that it should be removed.

**Comment by cereallarceny** [comment]
Waiting on https://github.com/OpenMined/PyGrid/issues/627 until we can merge

**Comment by Benardi** [comment]
> Waiting on #627 until we can merge

Done.

**Pull request merged**